content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md

```diff
@@ -67,11 +66,7 @@ Now that you have defined a stream, you can deploy it. To do so:
 
 If you are using the local Data Flow Server, add the following deployment property to set the port to avoid a port collision.
 
-**TODO show in UI**
-
-```
-stream deploy http-ingest --properties app.http.server.port=9000
-```
+    ![Unique Port](images/dataflow-unique-port.png)
```

Comment:
The indentation is showing this a code segment preventing image from showing up.